en/04_Changelogs/5.1.0.md

```diff
@@ -144,6 +145,7 @@ SilverStripe\ORM\DataList:
 - [`DataList::filterAny()`](api:SilverStripe\ORM\DataList::filterAny()) queries on many-many relations that use an aggregate `HAVING` clause now correctly use an `OR` conjunction rather than an incorrect `AND` conjunction.
 - At some point shortly before the release of Silverstripe CMS 4.0.0, SSL support for database connections was accidentally removed. This has now been reinstated - see [Using SSL in database connections](/developer_guides/security/secure_coding#using-ssl-in-database-connections) for more information.
 - The `cascade_duplicates` property was added to the [`InheritedPermissionsExtension`](api:SilverStripe\Security\InheritedPermissionsExtension) class so that now when duplicating any object that has the `InheritedPermissionsExtension` applied, the `GroupID` values in the `ViewerGroups` and `EditGroups` mapping tables will also be duplicated so that new object retains the same viewer and editor groups as the original.
+- And fields added to a model's `$summary_fields` configuration which are _not_ backed by database fields (such as method calls) will no longer be pulled through when `searchableFields()` calls back on it (i.e. because `$searchable_fields` configuration has not been explicitly declared). This means you do not need to explicitly declare `$searchable_fields` for models which should only use the summary fields to filter by.
```

Comment:
```suggestion
- Any fields added to a model's `$summary_fields` configuration which are _not_ backed by database fields (such as method calls) will no longer be pulled through when `searchableFields()` calls back on it (i.e. because `$searchable_fields` configuration has not been explicitly declared). This means you do not need to explicitly declare `$searchable_fields` for models which should only use the summary fields to filter by.
```

Reply:
Done